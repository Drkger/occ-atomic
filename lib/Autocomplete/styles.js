import spacing from '../subatomic/spacing';

export default {
    autoComplete: {
        position: 'relative'
    },
    droplist: {
        position: 'absolute',
        zIndex: 2,
        top: `calc(100% - ${spacing.xTiny}px)`,
        left: 0,
        width: '100%'
    }
};
