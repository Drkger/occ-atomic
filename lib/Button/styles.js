import colors from '../subatomic/colors';
import fonts from '../subatomic/fonts';
import icons from '../subatomic/icons';

export default {
    btn: {
        alignItems:'flex-start',
        boxSizing:'border-box',
        display:'inline-block',
        marginBottom:'0',
        textAlign:'center',
        whiteSpace:'nowrap',
        verticalAlign:'middle',
        touchAction:'manipulation',
        cursor:'pointer',
        userSelect:'none',
        padding:'0 30px',
        height:'30px',
        lineHeight:'24px',
        fontFamily:fonts.body,
        fontSize:'14px',
        fontWeight:'600',
        position:'relative',
        transition:'0.3s all',
        maxWidth:'100%',
        borderRadius:'5px',
        textDecoration:'none',
        '&:focus': {
            outline:'none'
        },
        '&:hover': {
            textDecoration:'none'
        }
    },
    cont: {
        display:'flex',
        width:'100%',
        height:'100%',
        alignItems:'center',
        justifyContent:'center',
        pointerEvents:'none'
    },
    loadCont: {
        display:'flex',
        width:'100%',
        height:'100%',
        alignItems:'center',
        justifyContent:'center',
        position:'absolute',
        top:'0',
        left:'0',
        pointerEvents:'none'
    },
    // Themes
    main: {
        background:colors.pink,
        color:colors.white,
        border:`1px solid ${colors.pink}`,
        '&:hover': {
            background:colors.pinkH,
            color:colors.white,
            borderColor:colors.pinkH
        },
        '&:active': {
            background:colors.pinkA,
            color:colors.white,
            borderColor:colors.pinkA
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&$loading': {
            background:colors.pinkA,
            borderColor:colors.pinkA
        },
        '&$disabled': {
            background:`${colors.grey1} !important`,
            color:`${colors.white} !important`,
            border:`1px solid ${colors.grey1} !important`
        }
    },
    secondary: {
        background:'transparent',
        color:colors.pink,
        border:`1px solid ${colors.pink}`,
        '&:hover': {
            background:colors.pinkH,
            color:colors.white,
            borderColor:colors.pinkH,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '&:active': {
            background:colors.pinkA,
            color:colors.white,
            borderColor:colors.pinkA,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.pink])) : ''
        },
        '&$loading': {
            background:colors.pinkA,
            borderColor:colors.pinkA
        },
        '&$disabled': {
            background:`${colors.grey1} !important`,
            color:`${colors.white} !important`,
            border:`1px solid ${colors.grey1} !important`
        }
    },
    white: {
        background:'transparent',
        color:colors.white,
        border:`1px solid ${colors.white}`,
        '&:hover': {
            background:'rgba(255,255,255,0.2)',
            color:colors.white,
            borderColor:colors.white,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '&:active': {
            background:'rgba(255,255,255,0.2)',
            color:colors.white,
            borderColor:colors.white,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&$loading': {
            background:'rgba(255,255,255,0.2)',
            borderColor:colors.white
        },
        '&$disabled': {
            background:`transparent !important`,
            color:`rgba(255,255,255,0.6) !important`,
            border:`1px solid rgba(255,255,255,0.6) !important`,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon(['rgba(255,255,255,0.6)'])) : ''
            }
        }
    },
    flat: {
        background:'transparent',
        color:colors.pink,
        border:`1px solid transparent`,
        '&:hover': {
            background:colors.grey3,
            color:colors.pink,
            borderColor:colors.grey3,
        },
        '&:active': {
            background:colors.grey5,
            color:colors.pink,
            borderColor:colors.grey5,
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.pink])) : ''
        },
        '& $loadIcon': {
            background: icons.base(icons.loading.icon([colors.pink]))
        },
        '&$loading': {
            background:colors.white,
            borderColor:colors.white
        },
        '&$disabled': {
            background:'transparent !important',
            color:`${colors.grey1} !important`,
            border:`1px solid transparent !important`,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.grey1])) : ''
            },
            '& $loadIcon': {
                background: icons.base(icons.loading.icon([colors.grey1]))
            },
        }
    },
    blue: {
        background:colors.blue,
        color:colors.white,
        border:`1px solid ${colors.blue}`,
        '&:hover': {
            background:colors.blueH,
            color:colors.white,
            borderColor:colors.blueH,
        },
        '&:active': {
            background:colors.blueA,
            color:colors.white,
            borderColor:colors.blueA,
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&$loading': {
            background:colors.blueA,
            borderColor:colors.blueA
        },
        '&$disabled': {
            background:`${colors.grey1} !important`,
            color:`${colors.white} !important`,
            border:`1px solid ${colors.grey1} !important`
        }
    },
    loading: {
        pointerEvents:'none',
        '& $cont': {
            visibility:'hidden'
        }
    },
    disabled: {
        background:colors.grey1,
        color:colors.white,
        border:`1px solid ${colors.grey1}`,
        cursor:'default',
        pointerEvents:'none',
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&:hover, &:active, &:focus': {
            background:colors.grey1,
            color:colors.white,
            border:`1px solid ${colors.grey1}`
        }
    },
    // Sizes
    md: {
        fontSize:'16px',
        height:'40px',
        lineHeight:'28px',
        '& i': {
            width:'28px',
            height:'28px'
        }
    },
    lg: {
        fontSize:'18px',
        height:'46px',
        lineHeight:'32px',
        '& i': {
            width:'32px',
            height:'32px'
        }
    },
    // Block
    block: {
        display:'block',
        width:'100%'
    },
    // Icon
    icon: {
        width:'24px',
        height:'24px',
        display:'inline-block',
        transition:'0.3s all'
    },
    '@keyframes rotate': {
        from: {transform: 'rotate(0deg)'},
        to: {transform: 'rotate(360deg)'}
    },
    loadIcon: {
        width:'24px',
        height:'24px',
        display:'inline-block',
        transition:'0.3s all',
        animation:'rotate 1s infinite linear',
        background: icons.base(icons.loading.icon())
    }
};