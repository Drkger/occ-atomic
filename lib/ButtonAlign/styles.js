export default {
    sm: {
        marginTop:'19px'
    },
    md: {
        marginTop:'9px'
    },
    lg: {
        marginTop:'3px'
    }
};