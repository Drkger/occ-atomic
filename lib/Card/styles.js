import colors from '../subatomic/colors';
import fonts from '../subatomic/fonts';
import shadows from '../subatomic/shadows';

const styles = {
    card: {
        background:colors.white,
        borderRadius:'5px',
        padding:'30px 20px'
    },
    cardNoPadding: {
        extend: 'card',
        padding:'0'
    }
};

const pushShadows = function() {
    for (let i = 0; i<7; i++) {
        styles[`shadow${i}`] = {
            boxShadow: shadows[`lvl${i}`]
        };
    }
};

pushShadows();

export default styles;