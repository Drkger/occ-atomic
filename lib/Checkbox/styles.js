import colors from '../subatomic/colors';
import spacing from '../subatomic/spacing';
import icons from '../subatomic/icons';

export default {
    cont: {
        paddingTop:spacing.tiny,
        paddingBottom:spacing.tiny,
        boxSizing:'border-box',
        display:'flex',
        alignItems:'center',
        cursor:'pointer',
        outline:'0',
        '&:after': {
            content:'""',
            display:'table',
            clear:'both'
        },
        '&:hover $check:after': {
            background:icons.base(icons.check.icon([colors.grey200]))
        }
    },
    check: {
        width:spacing.base,
        height:spacing.base,
        position:'relative',
        '&:before': {
            content:'""',
            width:spacing.small,
            height:spacing.small,
            borderRadius:4,
            position:'absolute',
            top:'50%',
            left:'50%',
            transform:'translate(-50%, -50%)',
            border:`1px solid ${colors.grey200}`,
            background:colors.bgWhite
        },
        '&:after': {
            content:'""',
            width:18,
            height:18,
            marginTop:-1,
            position:'absolute',
            top:'50%',
            left:'50%',
            transform:'translate(-50%, -50%)',
            transition:'0.3s all',
            background:'transparent'
        }
    },
    active: {
        '& $check:after': {
            background:[icons.base(icons.check.icon([colors.prim])), '!important']
        }
    },
    disabled: {
        opacity:0.4,
        pointerEvents:'none'
    },
    label: {
        marginLeft:spacing.tiny,
        cursor:'pointer'
    }
};
