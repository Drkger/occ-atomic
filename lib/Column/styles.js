import grid from '../subatomic/grid';

const styles = {
    col: {
        boxSizing:'border-box',
        position:'relative',
        minHeight:'1px',
        paddingRight:'15px',
        paddingLeft:'15px'
    }
};

const pushGrid = function() {
    const sm = `@media (min-width: ${grid.sm}px)`;
    const md = `@media (min-width: ${grid.md}px)`;
    const lg = `@media (min-width: ${grid.lg}px)`;
    for (let i = 1; i <= 12; i++) {
        styles[`xs${i}`] = {
            float:'left',
            width:`${100/12*i}%`
        };
        styles[`offset-xs${i}`] = {
            marginLeft:`${100/12*i}%`
        };
        styles[`push-xs${i}`] = {
            left:`${100/12*i}%`
        };
        styles[`pull-xs${i}`] = {
            right:`${100/12*i}%`
        };
    };
    styles[sm] = {};
    styles[md] = {};
    styles[lg] = {};
    for(let i = 1; i <= 12; i++) {
        styles[sm][`sm${i}`] = {
            float:'left',
            width:`${100/12*i}%`
        };
        styles[sm][`offset-sm${i}`] = {
            marginLeft:`${100/12*i}%`
        };
        styles[sm][`push-sm${i}`] = {
            left:`${100/12*i}%`
        };
        styles[sm][`pull-sm${i}`] = {
            right:`${100/12*i}%`
        };
        styles[md][`md${i}`] = {
            float:'left',
            width:`${100/12*i}%`
        };
        styles[md][`offset-md${i}`] = {
            marginLeft:`${100/12*i}%`
        };
        styles[md][`push-md${i}`] = {
            left:`${100/12*i}%`
        };
        styles[md][`pull-md${i}`] = {
            right:`${100/12*i}%`
        };
        styles[lg][`lg${i}`] = {
            float:'left',
            width:`${100/12*i}%`
        };
        styles[lg][`offset-lg${i}`] = {
            marginLeft:`${100/12*i}%`
        };
        styles[lg][`push-lg${i}`] = {
            left:`${100/12*i}%`
        };
        styles[lg][`pull-lg${i}`] = {
            right:`${100/12*i}%`
        };
    }
};

pushGrid();

export default styles;