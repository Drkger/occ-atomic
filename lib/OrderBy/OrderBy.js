import React from 'react';
import PropTypes from 'prop-types';

/** The OrderBy component shows a label and a dropdown with the values to order a set of items. You have to do that ordering by receiving the OrderBy value with an onChange function. */
class OrderBy extends React.Component {
    
    constructor(props) {
        super(props);
        this.onChange = this.onChange.bind(this);
    }
    
    onChange({target: {value}}) {
        const { onChange } = this.props;
        if (onChange) onChange(value);
    }
    
    render() {
        const { classes, text, options, value, id, className, style } = this.props;
        return (
            <div className={`${classes.wrap}${className ? ` ${className}` : ''}`} id={id} style={style}>
                <span className={classes.text}>{text}</span>
                <select className={classes.select} value={value} onChange={this.onChange}>
                    {options.map(option => (
                        <option key={option.value} value={option.value}>{option.text}</option>
                    ))}
                </select>
            </div>
        );
    }
}

OrderBy.propTypes = {
    classes: PropTypes.object,
    /** Text to be displayed on the left. */
    text: PropTypes.string,
    /** Array with all the options to show in the dropdown. */
    options: PropTypes.array.isRequired,
    /** Set a default value. */
    value: PropTypes.node,
    /** Function to call when the value of the dropdown changes. */
    onChange: PropTypes.func,
    id: PropTypes.string,
    className: PropTypes.string,
    style: PropTypes.object
};

export default OrderBy;