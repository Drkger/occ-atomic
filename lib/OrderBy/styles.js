import spacing from '../subatomic/spacing';

export default {
    wrap: {
        display:'inline-block',
        position:'relative'
    },
    button: {
        cursor:'pointer'
    },
    icon: {
        marginLeft:spacing.xTiny,
        marginTop:1
    },
    cardWrap: {
        position:'absolute',
        top:'100%',
        right:0,
        opacity:0,
        marginTop:-spacing.small,
        transition:'0.3s all',
        pointerEvents:'none'
    },
    card: {
        padding:`${spacing.tiny}px ${spacing.small}px`
    },
    show: {
        opacity:1,
        marginTop:0,
        pointerEvents:'auto'
    },
    option: {
        cursor:'pointer',
        whiteSpace:'nowrap'
    }
};
