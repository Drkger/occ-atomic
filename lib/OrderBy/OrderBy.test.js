import React from 'react';
import { shallow, mount } from 'enzyme';
import OrderBy from './OrderBy';
import styles from './styles';
import OrderByJSS from './index';

const reduceClasses = (prev, curr) => Object.assign({}, prev, { [curr]: curr });

const classes = Object.keys(styles).reduce(reduceClasses, {});

const options = [
    {
        value: 1,
        text: 'Option 1'
    },
    {
        value: 2,
        text: 'Option 2,'
    }
];

describe("OrderBy", () => {

    it('matches the snapshot', () => {
        const wrapper = shallow(
            <OrderBy classes={classes} options={options} />
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('shows the text', () => {
        const wrapper = shallow(
            <OrderBy classes={classes} options={options} text="Text" />
        );
        expect(wrapper.find('span').text()).toEqual('Text');
    });

    it('renders all the options', () => {
        const wrapper = shallow(
            <OrderBy classes={classes} options={options} />
        );
        expect(wrapper.find('option').length).toBe(2);
    });

    it('sets a value', () => {
        const wrapper = shallow(
            <OrderBy classes={classes} options={options} value={2} />
        );
        expect(wrapper.find('select').props().value).toBe(2);
    });

    it('calls onChange function', () => {
        const onChange = jest.fn();
        const wrapper = mount(
            <OrderBy classes={classes} options={options} onChange={onChange} />
        );
        wrapper.find('select').simulate('change');
        expect(onChange.mock.calls.length).toBe(1);
    });

});

describe("OrderByJSS", () => {

    it('matches the snapshot', () => {
        const wrapper = shallow(
            <OrderByJSS options={options} />
        );
        expect(wrapper).toMatchSnapshot();
    });

});

describe("OrderBy styles", () => {

    it('matches the snapshot', () => {
        expect(styles).toMatchSnapshot();
    });

});