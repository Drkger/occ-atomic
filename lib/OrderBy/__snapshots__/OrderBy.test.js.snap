// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderBy matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderBy
    classes={
      Object {
        "select": "select",
        "text": "text",
        "wrap": "wrap",
      }
    }
    options={
      Array [
        Object {
          "text": "Option 1",
          "value": 1,
        },
        Object {
          "text": "Option 2,",
          "value": 2,
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span
          className="text"
        />,
        <select
          className="select"
          onChange={[Function]}
          value={undefined}
        >
          <option
            value={1}
          >
            Option 1
          </option>
          <option
            value={2}
          >
            Option 2,
          </option>
        </select>,
      ],
      "className": "wrap",
      "id": undefined,
      "style": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "text",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <option
              value={1}
            >
              Option 1
            </option>,
            <option
              value={2}
            >
              Option 2,
            </option>,
          ],
          "className": "select",
          "onChange": [Function],
          "value": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "host",
            "props": Object {
              "children": "Option 1",
              "value": 1,
            },
            "ref": null,
            "rendered": "Option 1",
            "type": "option",
          },
          Object {
            "instance": null,
            "key": "2",
            "nodeType": "host",
            "props": Object {
              "children": "Option 2,",
              "value": 2,
            },
            "ref": null,
            "rendered": "Option 2,",
            "type": "option",
          },
        ],
        "type": "select",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span
            className="text"
          />,
          <select
            className="select"
            onChange={[Function]}
            value={undefined}
          >
            <option
              value={1}
            >
              Option 1
            </option>
            <option
              value={2}
            >
              Option 2,
            </option>
          </select>,
        ],
        "className": "wrap",
        "id": undefined,
        "style": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "text",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <option
                value={1}
              >
                Option 1
              </option>,
              <option
                value={2}
              >
                Option 2,
              </option>,
            ],
            "className": "select",
            "onChange": [Function],
            "value": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "host",
              "props": Object {
                "children": "Option 1",
                "value": 1,
              },
              "ref": null,
              "rendered": "Option 1",
              "type": "option",
            },
            Object {
              "instance": null,
              "key": "2",
              "nodeType": "host",
              "props": Object {
                "children": "Option 2,",
                "value": 2,
              },
              "ref": null,
              "rendered": "Option 2,",
              "type": "option",
            },
          ],
          "type": "select",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`OrderBy styles matches the snapshot 1`] = `
Object {
  "select": Object {
    "appearance": "none",
    "background": "url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgogICAgICAgICAgICAJIHZpZXdCb3g9IjAgMCAxMiAxMiIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMTIgMTI7IiB4bWw6c3BhY2U9InByZXNlcnZlIj4KICAgICAgICAgICAgPHBhdGggZmlsbD0iIzE0NzZmYiIgZD0iTTYsMy42aDMuMkM5LjcsMy42LDEwLDQsMTAsNC40YzAsMC4yLTAuMSwwLjQtMC4yLDAuNUw4LjYsNi4xbC0yLDJjLTAuMywwLjMtMC44LDAuMy0xLjEsMGwwLDBsLTItMkwyLjIsNQogICAgICAgICAgICAJQzEuOSw0LjcsMS44LDQuMiwyLjEsMy45YzAuMi0wLjIsMC40LTAuMywwLjYtMC4zQzIuNywzLjYsNiwzLjYsNiwzLjZ6Ii8+CiAgICAgICAgICAgIDwvc3ZnPg==)",
    "backgroundPosition": "calc(100% - 2px) 5px",
    "backgroundSize": "12px 12px",
    "border": "0",
    "color": "#727272",
    "cursor": "pointer",
    "fontFamily": "'Open Sans', sans-serif",
    "fontSize": "14px",
    "marginLeft": "5px",
    "outline": "0",
    "paddingRight": "25px",
    "textAlign": "right",
  },
  "text": Object {
    "color": "#727272",
    "fontFamily": "'Open Sans', sans-serif",
    "fontSize": "14px",
  },
  "wrap": Object {
    "display": "flex",
  },
}
`;

exports[`OrderByJSS matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Jss(OrderBy)
    options={
      Array [
        Object {
          "text": "Option 1",
          "value": 1,
        },
        Object {
          "text": "Option 2,",
          "value": 2,
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "select": "OrderBy-select-0-1-3",
        "text": "OrderBy-text-0-1-2",
        "wrap": "OrderBy-wrap-0-1-1",
      },
      "options": Array [
        Object {
          "text": "Option 1",
          "value": 1,
        },
        Object {
          "text": "Option 2,",
          "value": 2,
        },
      ],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "select": "OrderBy-select-0-1-3",
          "text": "OrderBy-text-0-1-2",
          "wrap": "OrderBy-wrap-0-1-1",
        },
        "options": Array [
          Object {
            "text": "Option 1",
            "value": 1,
          },
          Object {
            "text": "Option 2,",
            "value": 2,
          },
        ],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
